modules/cluster/templates/startup.sh.tpl

```diff
@@ -165,21 +172,6 @@ EOF
 chmod 644 /etc/profile.d/vault.sh
 source /etc/profile.d/vault.sh
 
-if [ ${internal_lb} != true ]; then
-  # Add health-check proxy because target pools don't support HTTPS
-  apt-get install -yqq nginx
-
-  cat <<EOF > /etc/nginx/sites-available/default
-server {
-  listen ${vault_proxy_port};
-  location / {
-    proxy_pass $VAULT_ADDR/v1/sys/health?uninitcode=200;
-  }
-}
-EOF
-  systemctl enable nginx
-  systemctl restart nginx
-fi
```

Comment:
Previously, nginx was used to redirect the google_compute_http_health_check, which always made a request for `/`.

Today, the `google_compute_http_health_check` is able to request a custom path, so nginx is no longer necessary.